bootstrap.sh

```diff
@@ -1,54 +1,67 @@
-#!/bin/bash
+#!/usr/bin/bash
 
-set -eux pipefail
+set -e  # Exit with nonzero exit code if anything fails
 
 echo "Validate sudo credentials ..."
 sudo -v
```

Comment:
I think we should just attempt to install sudo here, worst case we dont install it.

Reply:
Currently testing in newest push

Reply:
?

Reply:
my mistake, I thought that `sudi -v` was to validate that sudo was installed, not to input sudo credentials.